backend/BattleShipAPI/BattleShipAPI/Hubs/GameHub.cs

```diff
@@ -868,5 +808,17 @@ await _notificationService.NotifyClient(
                     gameRoom.ShipsConfig);
             }
         }
+
+public async Task NotifyClient(string method, params object[] args)
+{
+    await _notificationService.NotifyClient(Clients, Context.ConnectionId, method, args);
+}
+
+
+        public async Task NotifyGroup(string groupName, string method, params object[] args)
+        {
+            await _notificationService.NotifyGroup(Clients, groupName, method, args);
+        }
+
```

Comment:
Whats this doing here? Can we use backend/BattleShipAPI/BattleShipAPI/Notifications/NotificationService.cs that has same methods?